lib/RGModels.js

```diff
@@ -0,0 +1,50 @@
+/**
+ *
+ * A result model for finding the best harvesting tool including the tool if found and the digTime for that tool/block combo.
+ * The digTime will be Infinity if the block is not diggable with any tool the bot has.
+ *
+ * @param  {Item|null} tool The Item best suited to dig the block.  Can be null if no tool is needed or if item is not diggable.
+ * @param  {number} digTime Time in seconds to dig the block, Infinity if not diggable
+ */
+const BestHarvestTool = class {
```

Comment:
Is this syntax typical/common? I think `class BestHarvestTool {` may be more readable

Reply:
It's a way to define classes introduced in ES6.  Subtle differences aside, its basically preference of this form `class expression` vs `class declaration`